thedmstoolkit/character_generation.py

```diff
@@ -0,0 +1,369 @@
+            "Satyr",
+            "Yuan-Ti",
+            "Shifter",
+            "Tabaxi",
+            "Tiefling",
+            "Triton",
+            "Tortle",
+            "Vedalken",
+            "Warforged",
+            "Locathah",
+            "Bugbear",
+            "Goblin",
+            "Hobgoblin",
+            "Kobold",
+            "Lizardfolk",
+            "Orc",
+            "Dwarf",
+            "Elf",
+            "Gnome",
+            "Half-Elf",
+            "Half-Orc",
+            "Halfling",
+            "Human",
+        ],
+    }
+    ClassDict: dict[str, list] = {
+        "Martial": ["Fighter", "Monk", "Ranger", "Rogue"],
+        "Divine": ["Cleric", "Paladin", "Warlock"],
+        "Magic": ["Artificer", "Bard", "Druid", "Sorcerer", "Wizard"],
+        "All": [
+            "Fighter",
+            "Monk",
+            "Ranger",
+            "Rogue",
+            "Cleric",
+            "Paladin",
+            "Warlock",
+            "Artificer",
+            "Bard",
+            "Druid",
+            "Sorcerer",
+            "Wizard",
+        ],
+    }
+    AlignmentDict: dict[str, list] = {
+        "Good": ["Lawful Good", "Neutral Good", "Chaotic Good"],
+        "Neutral": ["Lawful Neutral", "True Neutral", "Chaotic Neutral"],
+        "Evil": ["Lawful Evil", "Neutral Evil", "Chaotic Evil"],
+        "All": [
+            "Lawful Good",
+            "Neutral Good",
+            "Chaotic Good",
+            "Lawful Neutral",
+            "True Neutral",
+            "Chaotic Neutral",
+            "Lawful Evil",
+            "Neutral Evil",
+            "Chaotic Evil",
+        ],
+    }
+    MethodDict: dict[str, list] = {
+        "StandardArray": [15, 14, 13, 12, 10, 8],
+        "3d6": [
+            random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6),
+            random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6),
+            random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6),
+            random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6),
+            random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6),
+            random.randint(1, 6) + random.randint(1, 6) + random.randint(1, 6),
+        ],
+    }
+    BackgroundList = [
+        "Acolyte",
+        "Athlete",
+        "Barbarian Tribe Member",
+        "Charlatan",
+        "City Watch",
+        "Criminal",
+        "Clan Crafter",
+        "Entertainer",
+        "Faction Agent",
+        "Far Traveler",
+        "Fisher",
+        "Folk Hero",
+        "Gladiator",
+        "Hermit",
+        "Knight",
+        "Noble",
+        "Outlander",
+        "Pirate",
+        "Sage",
+        "Sailor",
+        "Soldier",
+        "Urchin",
+    ]
+
+    def get_Generators(self):
+        """
+        Gives the list of generator keys
+        Returns:
+            List: a list of all generator keys
+        """
+        return Character_Generator.GeneratorList.keys
```

Comment:
This can be switched to `self.GeneratorList.keys` also, keys is a method so it should be `self.GeneratorLists.keys()` I would recommend writing some tests for this pull request because some of this code may just crash.

Reply:
Got it I'll write some tests for it this week